cdc/log.cc

```diff
@@ -26,7 +26,7 @@
 #include <seastar/util/defer.hh>
 #include <seastar/core/thread.hh>
 
-#include "cdc/cdc.hh"
+#include "cdc/log.hh"
```

Comment:
Good patch. I would have also put this explanation of the purpose of the different files in a comment at the top of these files, not just in a commit message.

Reply:
Added explanations